projects/ontimize-web-ngx/src/lib/components/input/password-input/o-password-input.component.ts

```diff
@@ -24,7 +25,8 @@ export const DEFAULT_OUTPUTS_O_PASSWORD_INPUT = [
   encapsulation: ViewEncapsulation.None
 })
 export class OPasswordInputComponent extends OTextInputComponent implements OnInit {
-
+  public hide: boolean = true;
+  public showPasswordButton: boolean = false;
```

Comment:
You should use the @InputConverter decorator to parse the 'show-password-button' 'true|false|yes|no' string values